### PR TITLE
feat - adding statistics page sections

### PR DESCRIPTION
**Pull request by mah51**
Don't really know much about contributing so I thought I would try. I hope I haven't messed something up!

### CODE REVIEW

File: src/pages/templates/page-sections/testimonials/gridBlurredBackdrop.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  avatar: string;
+  index: number;
+}
+
+function TestmonialCard(props: TestimonialCardProps) {
+  const { name, role, content, avatar, index } = props;
+  return (
+    <Flex
+      boxShadow={'lg'}
+      width={'640px'}
```
Comment (hauptrolle) [inline src/pages/templates/page-sections/testimonials/gridBlurredBackdrop.tsx:67]: This hard coded with kinda breaks the responsiveness. Maybe we should use `maxW` and `w=full` to keep it responsive

**Review by hauptrolle — CHANGES_REQUESTED** [review]
Hey @mah51 thank you so much for your contribution. The templates look pretty sick!

I just have one small remark regarding the `with`. If you could fix this I can merge them 🥳

File: src/pages/templates/page-sections/testimonials/gridBlurredBackdrop.tsx (hunk 1 above)
Comment (mah51) [reply src/pages/templates/page-sections/testimonials/gridBlurredBackdrop.tsx:67]: Ah Yeah, of course. Sorry for merging both features into this same pull request, I didn't realise it updated straight from my main branch like that!

**Comment by mah51** [comment]
Hi, I've made the changes you requested. I also made the profile pictures pop into a column format on smaller devices, just so the text paragraph width isn't too small. Thanks!

**Review by hauptrolle — APPROVED** [review]
This is looking good! Will merge it! Thank you so much for your contribution!

**Comment by hauptrolle** [comment]
@all-contributors please add @mah51 for code

**Pull request merged**